config.py

```diff
@@ -0,0 +1,7 @@
+"""
+Application configuration settings
+
+"""
+DEBUG = True
```

Comment:
I would prefer DEBUG default to False. It's safer if something where to go wrong with a prod deployment to fail back to DEBUG = False.

Reply:
Ah... yes... I forgot to change that back to `False`.... :flushed: